lib/plistLib/main.js

```diff
@@ -127,7 +127,8 @@ define([
             return doc;
         },
         getType: function(value) {
-            if(value instanceof IntObject)
+            if(value instanceof IntObject
+                    || typeof value === 'number' && value === (value|0))
```

Comment:
> The value produced will always be the value of one of the two operand expressions."

I use that a lot, but that may be a bit bad for beginners to read, i.e.:
```
var myVal = input || 15;
```
to set a default etc.

It is maybe a bad habit, as I said, for beginners its cryptic. But then again, I'd look it up if I don't know what it does and then learn something more about the language. So a beginner with good manual-reading habits would learn.


Also, it's quite useful to keep a javascript console open for such cases. And, I want to point out that I recently created a bug with something similar (fixed it again). So yeah, even if you are used to it, it still gets you sometimes: https://github.com/graphicore/Atem-CPS/commit/3b8109ab1b529c7057880e265d15bda1dbebcf18